### PR TITLE
add support for new user configuration options

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: docs/data-sources/pg.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Read-Only:
 - **admin_username** (String)
 - **backup_hour** (String)
 - **backup_minute** (String)
+- **enable_ipv6** (String)
```
Comment (dmitrii-vasilev) [inline docs/data-sources/pg.md:98]: Hmm, shouldn't it be `Boolean`?

File: docs/data-sources/elasticsearch.md [hunk 2]
```diff
@@ -183,6 +182,7 @@ Read-Only:
 
 - **elasticsearch** (String)
 - **kibana** (String)
+- **prometheus** (String)
```
Comment (dmitrii-vasilev) [inline docs/data-sources/elasticsearch.md:185]: Hmm, shouldn't it be `Boolean`?

**Review by dmitrii-vasilev — CHANGES_REQUESTED** [review]
Looks good in general. But I'm confused by `String` types in the docs for fields which suppose to be `Boolean`. I only commented on a couple of them.

File: docs/data-sources/pg.md (hunk 1 above)
Comment (mhoffm-aiven) [reply docs/data-sources/pg.md:98]: This is a known issue: #620 , relates to https://github.com/aiven/terraform-provider-aiven/blob/3a8c53ea153010145c4967d3a889ad8fc94cb899/aiven/internal/uconf/user_config.go#L53 

Id think we should ignore it for now and fix the user_config generation at some point.

**Review by dmitrii-vasilev — APPROVED** [review]
LGTM

**Pull request merged**